google/cloud/security/common/gcp_type/resource_util.py

```diff
@@ -42,6 +43,11 @@ class ResourceUtil(object):
             'class': project.Project,
             'plural': 'Projects',
             'level': 1
+        },
+        resource.ResourceType.GROUP: {
+            'class': group.Group,
+            'plural': 'Groups',
+            'level': 4
```

Comment:
I think that was the hierarchy level, but I don't recall if I am even using it. We can revisit it later.